src/theme/foundations/textStyles.ts

```diff
@@ -47,20 +47,21 @@ export const textStyles = {
   header5: {
     fontFamily: 'heading',
     fontSize: '1rem',
-    letterSpacing: '0.02em',
+    letterSpacing: '0.04em',
     lineHeight: 'normal',
     color: 'body'
   },
   header6: {
     fontFamily: 'heading',
     fontSize: '0.875rem',
-    letterSpacing: '0.02em',
+    letterSpacing: '0.04em',
```

Comment:
Same here... I'm seeing 0.02 on the design system 
<img width="993" alt="image" src="https://user-images.githubusercontent.com/54227730/204621088-6d742d74-113d-4caf-bd8e-2c907de3ab93.png">

@nloureiro could you maybe help clarify here?

Reply:
[desktop] h1 h2 h3 h4 >>>0.04rem  
[Mobile] h1 h2 h3 h4 >>>0.03rem  

[desktop] + [mobile] h5 h6 >>>> 0.02rem

does it help?

Reply:
Fixed